src/pudl/workspace/setup.py

```diff
@@ -134,6 +134,9 @@ def derive_paths(pudl_in, pudl_out):
     ps['pudl_db'] = "sqlite:///" + str(pathlib.Path(
         ps['sqlite_dir'], 'pudl.sqlite'))
 
+    ps['censusdp1tract_db'] = "sqlite:///" + str(pathlib.Path(
+        ps['sqlite_dir'], 'censusdp1tract.sqlite'))
+
```

Comment:
Still looking forward to using importable settings. Someday. See issue #684

Reply:
+1